### ISSUE TITLE
Can't register org.bukkit.scoreboard.Team

### ISSUE THREAD

**Issue by WilliamQiufeng** [TheLimeGlass/Skellett#92]
Having an issue on Paper238 1.14 server.
```
[17:31:19 ERROR]: Error occurred while enabling Skellett v1.9.8 (Is it up to date?)
java.lang.IllegalArgumentException: Can't register org.bukkit.scoreboard.Team with the code name team because that name is already used by types.team
        at ch.njol.skript.registrations.Classes.registerClass(Classes.java:90) ~[?:?]
        at com.gmail.thelimeglass.Register.types(Register.java:690) ~[?:?]
        at com.gmail.thelimeglass.Skellett.onEnable(Skellett.java:205) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.14.4.jar:git-Paper-238]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338) ~[patched_1.14.4.jar:git-Paper-238]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:420) ~[patched_1.14.4.jar:git-Paper-238]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:467) ~[patched_1.14.4.jar:git-Paper-238]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:381) ~[patched_1.14.4.jar:git-Paper-238]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:474) ~[patched_1.14.4.jar:git-Paper-238]
        at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:290) ~[patched_1.14.4.jar:git-Paper-238]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876) ~[patched_1.14.4.jar:git-Paper-238]
        at java.lang.Thread.run(Thread.java:834) [?:?]```

[Full log](https://github.com/TheLimeGlass/Skellett/files/4292126/latest.log)

**Comment by TheLimeGlass** [comment]
Please provide a list of your addons.

**Comment by TheLimeGlass** [comment]
Fixed in 1.9.10
